go/degausser/degausser.go

```diff
@@ -13,6 +13,8 @@ import (
 // element node's `.innerText` property.
 // This does not take layout or styling into account.
 func HTMLToPlainText(htmlMarkup string) (string, error) {
+	htmlMarkup = strings.Replace(htmlMarkup, "<title/>", "<title></title>", 1)
+	htmlMarkup = strings.Replace(htmlMarkup, "<title />", "<title></title>", 1)
```

Comment:
Could probably use a regex here for some arbitrary number of whitespaces?
/<title\s*\/>/

Reply:
Done.